rfc-036-use-main-as-the-default-branch-name.md

```diff
@@ -0,0 +1,35 @@
+# Use `main` as the default branch name
+
+## Summary
+
+Git's `master` branch name has its origin in master-slave relationships. This is
+insensitive to those whose ancestors were once enslaved. Instead, we MUST use
+`main` as the default branch name for our internal repositories unless there's a
+more descriptive name to use and we SHOULD push our clients to allow us to do
+the same for their code.
+
+## Problem
+
+The `master` branch name (along with most other technological uses of that word)
+has its origin in master-slave relationships. It doesn't fit our ethics to
+perpetuate that language. It's an relatively small step, but by changing it we
+can take one of many steps needed to try to break the cycle of institutional
+racism.
+
+## Proposal
+
+Instead of calling the default branch of a Git repository `master` we SHOULD
+name it `main`. We MUST do this for all internal repositories, unless there's a
+more appropriate name for the default branch (such as `production`). We SHOULD
```

Comment:
On the point of "production" as an alternative name to "main". 

I don't intend to reignite a debate over branching models, whether it be one or many. The reality is that we do have many Rails projects that use multiple branches, usually in the form of a develop branch. I also believe that this is how all wordpress projects are set up by default.

I wonder if it would therefore be preferable to use "production" over "main" when used in conjunction with a "develop" branch. Is there a convention around the use of "main" that accounts for this situation that we could follow? Is this the done thing?

In short, do we prefer "Production/Develop" over "Main/Devleop"?

Reply:
I think it's okay to suggest some safe alternatives and point to the fact we need to make another decision on which one we want our default to be. 

My remaining concern is the next steps. As they are I think they risk us spending a lot of time changing our tooling and all _existing_ repositories (for example as a Friday task), only to make a subsequent decision that requires them all to be changed back from `main` to `release` (for example). I think the right thing to do is to stop using master everywhere but it feels more pragmatic to wait for the next RFC to be agreed on first. I think this RFC should be about not using master going forwards and the next RFC is about choosing the preferred name and taking corrective actions on past repos. 

> - Change all existing internal repositories to rename `master` to `main` or
  another more descriptive name
> - Change all template repositories to rename `master` to `main`

Reply:
I've changed the next steps:

- Change all existing internal repositories with single "main" branches to
  rename `master` to `main`
- Change all template repositories to rename `master` to `main`
- Open another RFC to settle on naming conventions for repositories with
  multiple "main" branches (such as `master` and `develop`)

Does that work better?